homeassistant/components/vesync/const.py

```diff
@@ -44,4 +45,5 @@
     "LAP-V201-AUSR": "Vital200S",  # Alt ID Model Vital200S
     "Vital100S": "Vital100S",
     "LAP-V102S-WUS": "Vital100S",  # Alt ID Model Vital100S,
+    "LUH-D301S-WEU": "LUH-D301S-WEU",
```

Comment:
Are you able to add the other models of humidifiers? Models are listed here: https://github.com/chuckdeal97/hass-core/blob/1b1634c68543974646af87a35ee76630f518bb1c/homeassistant/components/vesync/const.py